elements/rh-tooltip/rh-tooltip.ts

```diff
@@ -100,6 +104,75 @@ export class RhTooltip extends LitElement {
   async hide() {
     await this.#float.hide();
   }
+
+  #onKeydown = (event: KeyboardEvent): void => {
+    if (event.key === 'Escape') {
+      this.hide();
+    }
+  };
+
+  #handleInvokerSlotChange(): void {
+    const invokerSlot = this.shadowRoot?.querySelector<HTMLSlotElement>('#invoker');
+    if (!invokerSlot) {
+      return;
+    }
+
+    const assignedNodes = invokerSlot.assignedElements({ flatten: true });
+
+    const focusableSelector = `
+      a[href]:not([inert]):not([inert] *):not([tabindex^="-"]),
+      input:not([type="hidden"]):not([type="radio"]):not([inert]):not([inert] *):not([tabindex^="-"]):not(:disabled),
+      input[type="radio"]:not([inert]):not([inert] *):not([tabindex^="-"]):not(:disabled),
+      select:not([inert]):not([inert] *):not([tabindex^="-"]):not(:disabled),
+      textarea:not([inert]):not([inert] *):not([tabindex^="-"]):not(:disabled),
+      button:not([inert]):not([inert] *):not([tabindex^="-"]):not(:disabled)
+    `;
+
+    const findFocusableElement = (node: Element): HTMLElement | null => {
+      if (node.matches(focusableSelector)) {
+        return node as HTMLElement;
+      }
+
+      if (node.tagName.toLowerCase().startsWith('rh-')) {
+        setTimeout(() => {
+          const shadowFocusable = node.shadowRoot?.querySelector(focusableSelector);
+          shadowFocusable?.setAttribute('aria-describedby', this.#tooltipContentID);
```

Comment:
we shouldn't need to do this. 

When I tried the deploy preview, I found that:

1. Orca linux didn't read the tooltip content in any browser
2. VO Safari didn't read the tooltip content
3. VO Chrome read the tooltip content, but not as a part of the tooltip, i.e. in a separate announcement


I then added `aria-describedby` to the shadow span wrapping the invoker slot:
```html
<div id="container"
     style="${styleMap(styles)}"
     class="${classMap({ open,
                        'initialized': !!this.#initialized,
                         [on]: !!on,
                         [anchor]: !!anchor,
                         [alignment]: !!alignment })}">
  <div id="invoker"
       class="c"
       role="tooltip"
       aria-describedby="tooltip"
       aria-labelledby="tooltip">
    <slot id="invoker"></slot>
  </div>
  <div id="tooltip"
       class="c"
       aria-hidden="${String(!open) as 'true' | 'false'}">
    <slot name="content">${this.content}</slot>
  </div>
</div>
```



I found that:
1. orca on linux didn't read the tooltip
4. VO safari didn't read the tooltip
5. VO chrome read the tooltip

So in other words, using lightdom aria attributes didn't appear to me to have much positive effect, and using the shadow DOM attributes seems to work a bit better

Reply:
Now I realize my direction thus far was futile since reading `aria-describedby` / IDREF pairs from across the shadowdom / lightdom will not work without cross-root ARIA.

It's also clear that adding these ARIA + IDREF attributes in the shadowdom does little to nothing to aid accessibility.

So, to prioritize accessibility, I have changed the demos and added some docs around `<rh-tooltip>` accessibility.